core/templates/services/UpgradedServices.ts

```diff
@@ -538,6 +538,7 @@ import { WrittenTranslationsObjectFactory } from
 import { SolutionVerificationService } from
   // eslint-disable-next-line max-len
   'pages/exploration-editor-page/editor-tab/services/solution-verification.service';
+import { QuestionValidationService } from './question-validation.service';
```

Comment:
Don't add any new service to UpgradedServices. These changes are not required.

Reply:
Done.